src/renderer/components/dashboard/DashboardView.tsx

```
@@ -13,14 +13,16 @@ import { api } from '@renderer/api';
 import { createLogger } from '@shared/utils/logger';
+import { formatDistanceToNow } from 'date-fns';
+import { Command, FolderGit2, FolderOpen, GitBranch, Search, Settings } from 'lucide-react';
 import { useShallow } from 'zustand/react/shallow';
 
 const logger = createLogger('Component:DashboardView');
-import { formatDistanceToNow } from 'date-fns';
-import { Command, FolderGit2, FolderOpen, GitBranch, Search, Settings } from 'lucide-react';
 
 import type { RepositoryGroup } from '@renderer/types/data';
 
+import { RoiBlock } from './RoiBlock';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)